build_runner/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+- **BUG FIX**: Running the `build_runner` binary without arguments no longer
```

Comment:
Nit: please add `## 0.7.1+1`

Reply:
Oh and update pubspec please :)

I think this is probly worth a publish right away

Reply:
Done.